src/Gir.Tests/GenerationGtk2TestBase.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+using NUnit.Framework;
+
+namespace Gir.Tests
+{
+	[TestFixture]
+	public abstract class GenerationGtk2TestBase
+	{
+		protected const string Gdk2 = "Gtk2.Gdk-2.0";
```

Comment:
Why not just add them into the generation test base?

Reply:
I started to do that, but the include directory as different. Might be worth merging them back together?

Reply:
We can probably infer the include directory from the name? i.e. `Gtk2.Gdk-2.0` -> include Gtk2, load Gdk-2.0.